src/monodroid/jni/mono_android_Runtime.h

```diff
@@ -39,6 +39,7 @@ JNIEXPORT void JNICALL Java_mono_android_Runtime_register
 JNIEXPORT void JNICALL Java_mono_android_Runtime_notifyTimeZoneChanged
   (JNIEnv *, jclass);
 
+#if !defined (ANDROID)
```

Comment:
I assume this was a *manual* addition?  This is usually a generated file, so if this is a manual addition that's…problematic?  It might be better to wrap the `#include "mono_android_Runtime.h"` instead.

Reply:
…though it's only the Desktop-related members here.  Ugh.

Reply:
might be a good opportunity to introduce the long-considered `DesignerRuntime` class, have the `native` methods declared *there*, and update the current Designer-centric `Runtime` methods to instead invoke the `DesignerRuntime` methods.  This means we'd have a *new* `mono_android_DesignerRuntime.h` file, which can be appropriately "guarded".

Reply:
Definitely not material for this PR :)